packages/cna-template/template/nuxt/package.json

```diff
@@ -3,7 +3,8 @@
     "dev": "nuxt",
     "build": "nuxt build",
     "start": "nuxt start",
-    "generate": "nuxt generate",
+    "export": "nuxt export",
```

Comment:
Should we conditionally add comments if target is static?

Reply:
I was thinking of it but afraid that renovate bot won't be working anymore, also it gives the possibility to easily switch targets by having all the comments.

Reply:
It can be done in `package.js` of same dir.